### PR TITLE
New: Parse Exception Groups QxR TAoE SMURF

### PR DESCRIPTION
**Pull request by bccornfo**
Update ExceptionReleaseGroupRegex with a number of release groups

#### Database Migration
NO

#### Description
This pull request aims to add a number of release groups that appear to follow the - RLSGROUP) format.

#### Todos
- [x] Tests
- [X] Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [X] [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes N/A

### CODE REVIEW

**Review by bakerboy448 — CHANGES_REQUESTED** [review]
Tests are required

the format of the exception group is not `-GROUP)` it is `GROUP)` or `GROUP]`.  Any group that's `-GROUP` is already recognized

**Comment by bccornfo** [comment]
> * Tests are required for parser changes. Please don't check the box indicating you've added tests when you did not.
> 
>     * This change breaks existing tests...QxR is not the group.
>       `should_parse_exception_release_group("Movie Name (2003) (2160p BluRay X265 HEVC 10bit HDR AAC 7.1 Tigole) [QxR]","Tigole")`
> 
> 
> the format of the exception group is not `-GROUP)` it is `GROUP)` or `GROUP]`. Any group that's `-GROUP` is already recognized

Added tests for the groups added to the ExceptionReleaseGroupRegex.

File: src/NzbDrone.Core.Test/ParserTests/ReleaseGroupParserFixture.cs [hunk 1]
```diff
@@ -106,7 +106,13 @@ public void should_parse_expected_release_group(string title, string expected)
         [TestCase("Movie Title (2009) (2160p PMTP WEB-DL Hybrid H265 DV HDR10+ DDP Atmos 5.1 English - HONE)", "HONE")]
         [TestCase("Why.Cant.You.Use.Normal.Characters.2021.2160p.UHD.HDR10+.BluRay.TrueHD.Atmos.7.1.x265-ZØNEHD", "ZØNEHD")]
         [TestCase("Movie.Should.Not.Use.Dots.2022.1080p.BluRay.x265.10bit.Tigole", "Tigole")]
-        [TestCase("Movie.Title.2005.2160p.UHD.BluRay.TrueHD 7.1.Atmos.x265 - HQMUX", "HQMUX")]
+        [TestCase("Movie.Title.2005.2160p.UHD.BluRay.TrueHD 7.1.Atmos.x265 - HQMUX", "HQMUX")]        
+        [TestCase("Movie Title (2022) (2160p ATV WEB-DL Hybrid H265 DV HDR DDP Atmos 5.1 English - SMURF)", "SMURF")]
+        [TestCase("Movie Title (2009) (2160p PMTP WEB-DL Hybrid H265 DV HDR10+ DDP Atmos 5.1 English - SMURF)", "SMURF")]        
+        [TestCase("Movie Title (2022) (2160p ATV WEB-DL x265 SDR DDP 5.1 English - JBENT TAoE)", "TAoE")]
```
Comment (bakerboy448) [inline src/NzbDrone.Core.Test/ParserTests/ReleaseGroupParserFixture.cs:112]: Group is JBENT not TAOE, no?

Comment (bccornfo) [reply src/NzbDrone.Core.Test/ParserTests/ReleaseGroupParserFixture.cs:112]: No, the relase group is TAoE

File: src/NzbDrone.Core.Test/ParserTests/ReleaseGroupParserFixture.cs [hunk 2]
```diff
@@ -106,7 +106,13 @@ public void should_parse_expected_release_group(string title, string expected)
...
+        [TestCase("Movie Title (2009) (2160p PMTP WEB-DL x265 SDR DDP 5.1 English - JBENT TAoE) ", "TAoE")]        
+        [TestCase("Movie Title (2022) (2160p HMAX WEB-DL x265 SDR DDP Atmos 5.1 English - Ghost QxR)", "QxR")]
+        [TestCase("Movie Title (2009) (2160p NF WEB-DL x265 SDR DDP Atmos 5.1 English - Ghost QxR)", "QxR")]
```
Comment (bakerboy448) [inline src/NzbDrone.Core.Test/ParserTests/ReleaseGroupParserFixture.cs:115]: Group is ghost, not qxr

File: src/NzbDrone.Core.Test/ParserTests/ReleaseGroupParserFixture.cs (hunk 1 above)
Comment (bccornfo) [reply src/NzbDrone.Core.Test/ParserTests/ReleaseGroupParserFixture.cs:112]: The group member is JBENT

File: src/NzbDrone.Core.Test/ParserTests/ReleaseGroupParserFixture.cs (hunk 2 above)
Comment (bccornfo) [reply src/NzbDrone.Core.Test/ParserTests/ReleaseGroupParserFixture.cs:115]: QxR is the group, ghost is the member

File: src/NzbDrone.Core.Test/ParserTests/ReleaseGroupParserFixture.cs [hunk 3]
```diff
@@ -106,7 +106,13 @@ public void should_parse_expected_release_group(string title, string expected)
         [TestCase("Movie Title (2009) (2160p PMTP WEB-DL Hybrid H265 DV HDR10+ DDP Atmos 5.1 English - HONE)", "HONE")]
         [TestCase("Why.Cant.You.Use.Normal.Characters.2021.2160p.UHD.HDR10+.BluRay.TrueHD.Atmos.7.1.x265-ZØNEHD", "ZØNEHD")]
         [TestCase("Movie.Should.Not.Use.Dots.2022.1080p.BluRay.x265.10bit.Tigole", "Tigole")]
-        [TestCase("Movie.Title.2005.2160p.UHD.BluRay.TrueHD 7.1.Atmos.x265 - HQMUX", "HQMUX")]
+        [TestCase("Movie.Title.2005.2160p.UHD.BluRay.TrueHD 7.1.Atmos.x265 - HQMUX", "HQMUX")]
```
Comment (Qstick) [inline src/NzbDrone.Core.Test/ParserTests/ReleaseGroupParserFixture.cs:109]: Get rid of trailing space in multiple areas

```suggestion
        [TestCase("Movie.Title.2005.2160p.UHD.BluRay.TrueHD 7.1.Atmos.x265 - HQMUX", "HQMUX")]
```

**Review by Qstick — CHANGES_REQUESTED** [review]

File: src/NzbDrone.Core.Test/ParserTests/ReleaseGroupParserFixture.cs (hunk 3 above)
Comment (bccornfo) [reply src/NzbDrone.Core.Test/ParserTests/ReleaseGroupParserFixture.cs]: Done

**Comment by grapexy** [comment]
Just wanted to chime in here before/if this gets merged: TAoE and QxR are not 1:1 mapping to groups, they're more akin to a super-group, or an organization. Their releases differ greatly and frequently do not have this super-group in filenames or even release names sometimes, but they always have their "member" name. So for purposes of *arrs, "Tigole" and "Goki" should be a parsed group, not "QxR" or "TAoE".

**Comment by bakerboy448** [comment]
needs to be rebased & deconflicted

**Pull request closed without merging**